### ISSUE TITLE
Handle case where there is only 1 greeter

### ISSUE THREAD

**Issue by ilmari-lauhakangas** [qarkai/WelcomeBot#2]
Currently with 1 greeter the bot just says (like ).

This should be fixed with something like

```
if len(self.greeters) > 2:
    greeters_string += ", and {}".format(self.greeters[-1])
elif len(self.greeters) == 2:
    greeters_string += " and {}".format(self.greeters[-1])
else:
    greeters_string = "{}".format(self.greeters[-1])
```

**Comment by qarkai** [comment]
Fixed in 231ccdc04a14d212b84e45cb002d1766b78ded65.

**Comment by qarkai** [comment]
I've added check for empty greeters list and return empty string but there should be another reply message for this case. Otherwise bot would say '(like )' when no admins defined for channel in settings.
